### ISSUE TITLE
mandatory_scheduling tour choice fails if no time window for second tour

### ISSUE THREAD

**Issue by toliwaga** [UDST/activitysim#121]
mandatory_scheduling fails with no non-zero probability alternatives for both work and school tours if the first tour chosen is the last possible tour of the day and so all alternatives are -999

For now, I simply assigned tours starting before previous tour ends -100 isntead of -999

```
# FIXME - Subsequent tour must start after previous tour ends,(tour_num > 1) & (start < end_previous),-999
Subsequent tour must start after previous tour ends,(tour_num > 1) & (start < end_previous),-100
```

**Comment by toliwaga** [comment]
non_mandatory_scheduling has the same problem, and I am implementing the same patch in configs/tour_departure_and_duration_nonmandatory.csv until we have a more satisfactory solution.

**Comment by wusun2** [comment]
The CT-RAMP have the same issue.  We treated it this way:  if the 1st tour is scheduled to the last time period, all subsequent tours are scheduled to last time periods too.  By setting a util of -100, essentially you allowed scheduling 2nd or 3rd tours before the first tour.

**Comment by bstabler** [comment]
correct.  we're just parking this issue here for now until we correct these models.
